web/app/themes/wordpress-scaffold/lib/Grrr/Rest/Routes.php

```diff
@@ -0,0 +1,32 @@
+<?php namespace Grrr\Rest;
+
+use Garp\Functional as f;
+
+class Routes {
+
+    const NAMESPACE = 'grrr/v1';
+
+    const ROUTES = [
+        'newsletter' => 'newsletter/subscribe',
+    ];
+
+    public static function namespace(): string {
+        return static::NAMESPACE;
+    }
+
+    public static function get(string $name): string {
+        return f\prop($name, static::ROUTES);
+    }
+
+    public static function all(): array {
+        return f\reduce(function($endpoints, $endpoint) {
+            $endpoints[] = '/' . static::NAMESPACE . '/' . $endpoint;
+            return $endpoints;
+        }, [], static::ROUTES);
+    }
+
+    public static function url(string $name): string {
+        return rest_url('/' . endpoint($name));
```

Comment:
I don't know this `endpoint` function.

Reply:
Great spot, it doesn't exist. Fixed now 👍